index.html

```diff
@@ -972,64 +972,127 @@ <h2>UTF-8 Character Encoding Form</h2>
 </ul>
 </aside>
 
-<div class="xref"><span class="seealso">See also</span>
-<p>[[[#char_string]]].</p>
-</div>
-
-
 
 	<div class="req" id="char_specific">
 	<p class="advisement">Specifications SHOULD use specific terms, when available, instead of the general term 'character'.</p>
-	<details class="links"><summary>explanations &amp; examples</summary>
+	<details class="links"><summary>source</summary>
 	<p><a href="https://www.w3.org/TR/charmod/#sec-PerceptionsOutro">Perceptions of Characters, Summary C067</a>, in <cite>Character Model for the World Wide Web 1.0: Fundamentals</cite>.</p>
 	</details>
 	</div>
 
-	<div class="req" id="char_define">
+
+<p>Specific terms could include <a href="https://www.w3.org/TR/i18n-glossary/#def_code_point">code point</a>, <a href="https://www.w3.org/TR/i18n-glossary/#def_grapheme_cluster">grapheme cluster</a>, <a href="https://www.w3.org/TR/i18n-glossary/#def_code_unit">code unit</a>, and <a href="https://www.w3.org/TR/i18n-glossary/#def_glyph">glyph</a>.</p>
+
+
+<div class="req" id="char_define">
 	<p class="advisement">When specifications use the term 'character' the specifications MUST define which meaning they intend, and  SHOULD explicitly define the term 'character' to mean a Unicode code point.</p>
-	<details class="links"><summary>explanations &amp; examples</summary>
+	<details class="links"><summary>source</summary>
 	<p><a href="https://www.w3.org/TR/charmod/#sec-PerceptionsOutro">Perceptions of Characters, Summary C010</a>, in <cite>Character Model for the World Wide Web 1.0: Fundamentals</cite></p>
 	</details>
-	</div>
+  </div>
+
+<p>The developers of specifications, and the developers of software based on those specifications, are likely to be more familiar with usages of the term 'character' they have experienced and less familiar with the wide variety of usages in an international context. Furthermore, within a computing context, characters are often confused with related concepts, resulting in incomplete or inappropriate specifications and software.</p>
 
 	<div class="req" id="char_physical_storage">
 	<p class="advisement">Specifications, software and content MUST NOT require or depend on a one-to-one relationship between characters and units of physical storage.</p>
-	<details class="links"><summary>explanations &amp; examples</summary>
+	<details class="links"><summary>source</summary>
 	<p><a href="https://www.w3.org/TR/charmod/#sec-Storage">Units of storage C009</a>, in <cite>Character Model for the World Wide Web 1.0: Fundamentals</cite></p>
 	</details>
 	</div>
 
-	<div class="req" id="char_sounds">
+<p>Computer storage and communication rely on units of physical storage and information interchange, such as bits and bytes (8-bit units, also called octets). A frequent error in specifications and implementations is the equating of characters with units of physical storage. The mapping between characters and such units of storage is actually quite complex.</p>
```

Comment:
We'll probably need to do more work here. I note that the Section 4 intro talks about this somewhat. I realize that this is verbatim from charmod: my tendency is to want to use (and link to) _code unit_ here somehow...

Reply:
Yes, i was aware that a cross-reference would be needed here eventually, but left that for another day's edits.